### PR TITLE
Kcyganik fix tox - closes #96

### LINKED ISSUE #96: Make tox work with both mypy and flake8

### PR DESCRIPTION
**Pull request by kordc**

### CODE REVIEW

File: art/utils/LoggerAdapter.py
Comment (mmaecki) [inline art/utils/LoggerAdapter.py:1]: Looks fine

File: art/cli/__main__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from .main import app
+
+if __name__ == "__main__":
```
Comment (SebChw) [inline art/cli/__main__.py]: Not the very important thing but you actually don't need to write this if statement here https://docs.python.org/3/library/__main__.html#main-py-in-python-packages

**Comment by SebChw** [comment]
Well, done thanks for writing the stubs files 🥳

**Comment by SebChw** [comment]
Maybe I have one thing. Maybe we should add special requirments to install tox mypy etc. This should work like `pip install -e .[dev]

**Comment by SebChw** [comment]
I have error on windows with tox. This is just and windows issue?
image

**Comment by SebChw** [comment]
I wonder if this makes sense to use autopep8 and then flake8. I assume that autopep8 doesn't create PR automatically. This test will always pass then but won't have influence on the repository. I'd rather prefer obtaining exception from flake8 and someone to manually run autopep8.

File: tox.ini [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
-deps = mypy
-commands = python -m mypy art #Withouth python -m it cries about third-party libraries
-
-[testenv:flake8]
-deps = flake8
-commands = flake8 art
\ No newline at end of file
+commands =
+    pytest tests
+    - autopep8 --in-place --recursive art
```
Comment (SebChw) [inline tox.ini]: I wonder if this makes sense to use autopep8 and then flake8. I assume that autopep8 doesn't create PR automatically. This test will always pass then but won't have influence on the repository. I'd rather prefer obtaining exception from flake8 and someone to manually run autopep8.

to be discussed, but this is why I requested changes

**Review by SebChw — DISMISSED** [review]

File: tox.ini (hunk 2 above)
Comment (kordc) [reply tox.ini]: agree

File: art/cli/__main__.py (hunk 1 above)
Comment (kordc) [reply art/cli/__main__.py]: interesting, thanks

**Review by trebacz626 — APPROVED** [review]
Approve

**Pull request merged**